web/index.html

```diff
@@ -57,6 +50,12 @@ <h3 class="sub-header">
                 <div class="github-link-text">Star my repo!</div>
             </a>
 
+            <div v-if="!isMobile" class="import-button">
+                <div style="margin:0px" class="button" @click="showImportToast=true">
+                    Import Data
+                </div>
+            </div>
```

Comment:
added the import data button

Reply:
not visible on mobile via the v-if, but also hidden on smaller screen sizes as part of the import-button class